content/en/_index.md

```diff
@@ -1,41 +1,52 @@
-
-Development of the system moved from Xerox PARC to a Xerox AI Systems division, to a spin-out company called Envos, to a smaller company called Venue. The system was called Interlisp, Interlisp-D, and various named releases (Koto, Lyric, Medley) until the name "Medley" was used for the whole thing. [More background](about).
-
-## Project overview
-
-March 18, 2023 we presented an overview of the project to the BALISP group (Bay Area Lisp). 
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/N1MobfEaoWY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-
-## What are we trying to accomplish?
-
-
-We aim to revive Medley Interlisp to support not just a demo or test drive but actual use as a development and learning tool. To make the software usable, we need to overcome a number of compatibility problems with current systems and interfaces.
-
-We also want to restore and present earlier versions of Interlisp, for the student of computer history. See [Interlisp and Software Preservation Network](https://www.softwarepreservationnetwork.org/Interlisp/) for more.
-
-## What have we done so far?
-
-There has been a lot of cleanup and adaptation to make it usable again in the modern world. Among other developments, you can now run Medley Interlisp on many OS and hardware configurations, or at https://online.interlisp.org in the cloud, using a web browser. See our [2021 Annual Report](news/2021medleyannualreport), [Project News](/news/), and [recent release notes](https://github.com/Interlisp/medley/releases) . We've also been working on integration of the Interlisp style development with git and GitHub, Docker and other modern components.
-
-## Who is involved?
-
-We are some of the original developers and users of the system 30 years ago, joined by newcomers interested in software history and preservation, along with members of the Lisp community. We work with [organizational partners](about/partners). And with you! See [Get Involved](about/getinvolved/) for ways to get involved.
-
-## Testimonials
-
-[What have we lost?](https://www.youtube.com/watch?v=7RNbIEJvjUA&t=841s)  A presentation from the 2020 Remote Chaos Experience conference highlights some of the interesting aspects of Interlisp.  The presentation synopsis states it this way:
-> We have ended up in a world where UNIX and Windows have taken over, and most people have never experienced anything else. Over the years, though, many other system designs have come and gone, and some of those systems have had neat ideas that were nevertheless not enough to achieve commercial success. We will take you on a tour of a variety of those systems, talking about what makes them special.
+## Welcome to Interlisp.org
```

Comment:
Here's what the main page looks like with these changes.  I'm not a fan of the blue boxes.  Is this what we envision it looking like, or are these placeholders?  The boxes hyperlink to other locations in the document.

What's the hook that is going to keep someone that arrives at this site digging deeper?  We need some pictures or short videos that give a feel for what Medley is - maybe a couple pictures we cycle through with images of the desktop or short videos of a window being opened, pictures from an old conference.  Something that will appeal to readers.

![image](https://github.com/Interlisp/Interlisp.github.io/assets/22384842/b1a63c15-5c04-4d34-9f10-710ec4f43b12)

Reply:
I'm going to tackle this PR piece by piece.  With 75 files it's a bit of a beast.  I might suggest as we make progress on individual sections we break them into separate PRs and merge them.  That will, hopefully, keep the change management somewhat simplified and start moving new content to the website sooner.